src/main/java/rx/internal/operators/BackpressureUtils.java

```diff
@@ -45,7 +45,9 @@ private BackpressureUtils() {
      * @param n
      *            the number of requests to add to the requested count
      * @return requested value just prior to successful addition
+     * @deprecated Android has issues with reflection-based atomics
```

Comment:
It's one thing those devices can't find fields but how about reflection in general. Desktop Java does well with inlining and intrinsification of AtomicXFieldUpdaters; I'm not sure about Android in general through.